projects/github-actions/test-results-to-slack/src/slack.js

```diff
@@ -0,0 +1,57 @@
+const debug = require( './debug' );
+
+/**
+ * Sends a Slack message.
+ *
+ * @param {Object} client - Slack client
+ * @param {boolean} update - if it should update a message. For true, it will update an existing message based on `ts`, false will send a new message.
+ * @param {Object} options - options
+ */
+async function sendMessage( client, update, options ) {
+	const { text, blocks = [], channel, username, icon_emoji, ts, thread_ts } = options;
```

Comment:
Note that we are not checking the validity of the options passed. It is probably not a big deal atm, but if it would be used elsewhere, it could fail badly without us knowing it.

Reply:
Good point. I'll make a note for it for the next PR.